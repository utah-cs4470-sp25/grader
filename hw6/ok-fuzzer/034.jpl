show true

show (if (if (true && true) then (! false) else true) then ((- (if (689 >= 145) then 99.25 else (- 24.69))) / rgba {45.74, (if true then (- 43.79) else 58.92), (if (if false then false else true) then 69.9 else 59.02), 10.72}.b) else (if (false == (! true)) then 16.67 else rgba {(if true then 31.14 else 1.78), 69.06, 66.58, (if (1.86 == 89.01) then 27.38 else (- 15.64))}.g))

show ((if (! (80.52 < (49.05 - 84.95))) then (- (- (- (6.7 - 60.5)))) else (- 19.88)) % (if (([673][(- 213)] % 951) > 297) then (- 98.09) else (65.98 % rgba {(12.39 * 17.9), (if true then 1.65 else 49.09), 39.03, 47.78}.a)))

show 896

show (if (if false then (! ((- 70) >= 296)) else ((if true then (- 17.2) else 89.42) >= rgba {12.76, 42.08, 17.14, 29.08}.r)) then rgba {(if false then (if true then (56.92 * 37.12) else 13.78) else 43.0), (- (if (10.66 <= 77.56) then 39.51 else 21.15)), (82.14 - 26.98), (if ((258 + 537) != (587 + 251)) then 71.93 else 29.22)} else rgba {89.65, (11.87 % 69.49), 85.57, (- 55.49)}).g

show ((! ((if (true || (true && true)) then (147 + 349) else 605) > (- (523 * (if false then 685 else 426))))) || (if (26.61 > (- (if true then 69.82 else (- 74.22)))) then (false && (if (true == false) then ((- 43.04) == (- 86.15)) else ((97.47 == 82.67) == (false || true)))) else ((! (! false)) && (0.15 >= 15.32))))

struct a {
a : void
}
show ((- rgba {(- 96.09), ((- 5.63) % (12.73 % 81.81)), (- (- 8.34)), 91.89}.a) != 4.19)

struct b {
a : a
}
struct c {
a : float
b : float
c : int
}
struct d {
a : int
}
struct e {
a : float
b : c
}
struct f {
a : void
}
show 91.47

show e {(if (! (true && true)) then 94.9 else (- 41.35)), c {44.0, (- (97.19 / 16.13)), [d {65}][(- (- 371))].a}}

show (- [408][((if (! true) then (- 93) else 884) + 916)])

show ((436 != (if true then 774 else 774)) && (681 <= d {223}.a))

show ((e {48.79, c {(- 26.15), (21.33 % 8.49), 648}}.a + (- 54.43)) <= 55.12)

