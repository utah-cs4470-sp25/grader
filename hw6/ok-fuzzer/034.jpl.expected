(ShowCmd (TrueExpr (BoolType)))
(ShowCmd (IfExpr (FloatType) (IfExpr (BoolType) (BinopExpr (BoolType) (TrueExpr (BoolType)) && (TrueExpr (BoolType))) (UnopExpr (BoolType) ! (FalseExpr (BoolType))) (TrueExpr (BoolType))) (BinopExpr (FloatType) (UnopExpr (FloatType) - (IfExpr (FloatType) (BinopExpr (BoolType) (IntExpr (IntType) 689) >= (IntExpr (IntType) 145)) (FloatExpr (FloatType) 99) (UnopExpr (FloatType) - (FloatExpr (FloatType) 24)))) / (DotExpr (FloatType) (StructLiteralExpr (StructType rgba) rgba (FloatExpr (FloatType) 45) (IfExpr (FloatType) (TrueExpr (BoolType)) (UnopExpr (FloatType) - (FloatExpr (FloatType) 43)) (FloatExpr (FloatType) 58)) (IfExpr (FloatType) (IfExpr (BoolType) (FalseExpr (BoolType)) (FalseExpr (BoolType)) (TrueExpr (BoolType))) (FloatExpr (FloatType) 69) (FloatExpr (FloatType) 59)) (FloatExpr (FloatType) 10)) b)) (IfExpr (FloatType) (BinopExpr (BoolType) (FalseExpr (BoolType)) == (UnopExpr (BoolType) ! (TrueExpr (BoolType)))) (FloatExpr (FloatType) 16) (DotExpr (FloatType) (StructLiteralExpr (StructType rgba) rgba (IfExpr (FloatType) (TrueExpr (BoolType)) (FloatExpr (FloatType) 31) (FloatExpr (FloatType) 1)) (FloatExpr (FloatType) 69) (FloatExpr (FloatType) 66) (IfExpr (FloatType) (BinopExpr (BoolType) (FloatExpr (FloatType) 1) == (FloatExpr (FloatType) 89)) (FloatExpr (FloatType) 27) (UnopExpr (FloatType) - (FloatExpr (FloatType) 15)))) g))))
(ShowCmd (BinopExpr (FloatType) (IfExpr (FloatType) (UnopExpr (BoolType) ! (BinopExpr (BoolType) (FloatExpr (FloatType) 80) < (BinopExpr (FloatType) (FloatExpr (FloatType) 49) - (FloatExpr (FloatType) 84)))) (UnopExpr (FloatType) - (UnopExpr (FloatType) - (UnopExpr (FloatType) - (BinopExpr (FloatType) (FloatExpr (FloatType) 6) - (FloatExpr (FloatType) 60))))) (UnopExpr (FloatType) - (FloatExpr (FloatType) 19))) % (IfExpr (FloatType) (BinopExpr (BoolType) (BinopExpr (IntType) (ArrayIndexExpr (IntType) (ArrayLiteralExpr (ArrayType (IntType) 1) (IntExpr (IntType) 673)) (UnopExpr (IntType) - (IntExpr (IntType) 213))) % (IntExpr (IntType) 951)) > (IntExpr (IntType) 297)) (UnopExpr (FloatType) - (FloatExpr (FloatType) 98)) (BinopExpr (FloatType) (FloatExpr (FloatType) 65) % (DotExpr (FloatType) (StructLiteralExpr (StructType rgba) rgba (BinopExpr (FloatType) (FloatExpr (FloatType) 12) * (FloatExpr (FloatType) 17)) (IfExpr (FloatType) (TrueExpr (BoolType)) (FloatExpr (FloatType) 1) (FloatExpr (FloatType) 49)) (FloatExpr (FloatType) 39) (FloatExpr (FloatType) 47)) a)))))
(ShowCmd (IntExpr (IntType) 896))
(ShowCmd (DotExpr (FloatType) (IfExpr (StructType rgba) (IfExpr (BoolType) (FalseExpr (BoolType)) (UnopExpr (BoolType) ! (BinopExpr (BoolType) (UnopExpr (IntType) - (IntExpr (IntType) 70)) >= (IntExpr (IntType) 296))) (BinopExpr (BoolType) (IfExpr (FloatType) (TrueExpr (BoolType)) (UnopExpr (FloatType) - (FloatExpr (FloatType) 17)) (FloatExpr (FloatType) 89)) >= (DotExpr (FloatType) (StructLiteralExpr (StructType rgba) rgba (FloatExpr (FloatType) 12) (FloatExpr (FloatType) 42) (FloatExpr (FloatType) 17) (FloatExpr (FloatType) 29)) r))) (StructLiteralExpr (StructType rgba) rgba (IfExpr (FloatType) (FalseExpr (BoolType)) (IfExpr (FloatType) (TrueExpr (BoolType)) (BinopExpr (FloatType) (FloatExpr (FloatType) 56) * (FloatExpr (FloatType) 37)) (FloatExpr (FloatType) 13)) (FloatExpr (FloatType) 43)) (UnopExpr (FloatType) - (IfExpr (FloatType) (BinopExpr (BoolType) (FloatExpr (FloatType) 10) <= (FloatExpr (FloatType) 77)) (FloatExpr (FloatType) 39) (FloatExpr (FloatType) 21))) (BinopExpr (FloatType) (FloatExpr (FloatType) 82) - (FloatExpr (FloatType) 26)) (IfExpr (FloatType) (BinopExpr (BoolType) (BinopExpr (IntType) (IntExpr (IntType) 258) + (IntExpr (IntType) 537)) != (BinopExpr (IntType) (IntExpr (IntType) 587) + (IntExpr (IntType) 251))) (FloatExpr (FloatType) 71) (FloatExpr (FloatType) 29))) (StructLiteralExpr (StructType rgba) rgba (FloatExpr (FloatType) 89) (BinopExpr (FloatType) (FloatExpr (FloatType) 11) % (FloatExpr (FloatType) 69)) (FloatExpr (FloatType) 85) (UnopExpr (FloatType) - (FloatExpr (FloatType) 55)))) g))
(ShowCmd (BinopExpr (BoolType) (UnopExpr (BoolType) ! (BinopExpr (BoolType) (IfExpr (IntType) (BinopExpr (BoolType) (TrueExpr (BoolType)) || (BinopExpr (BoolType) (TrueExpr (BoolType)) && (TrueExpr (BoolType)))) (BinopExpr (IntType) (IntExpr (IntType) 147) + (IntExpr (IntType) 349)) (IntExpr (IntType) 605)) > (UnopExpr (IntType) - (BinopExpr (IntType) (IntExpr (IntType) 523) * (IfExpr (IntType) (FalseExpr (BoolType)) (IntExpr (IntType) 685) (IntExpr (IntType) 426)))))) || (IfExpr (BoolType) (BinopExpr (BoolType) (FloatExpr (FloatType) 26) > (UnopExpr (FloatType) - (IfExpr (FloatType) (TrueExpr (BoolType)) (FloatExpr (FloatType) 69) (UnopExpr (FloatType) - (FloatExpr (FloatType) 74))))) (BinopExpr (BoolType) (FalseExpr (BoolType)) && (IfExpr (BoolType) (BinopExpr (BoolType) (TrueExpr (BoolType)) == (FalseExpr (BoolType))) (BinopExpr (BoolType) (UnopExpr (FloatType) - (FloatExpr (FloatType) 43)) == (UnopExpr (FloatType) - (FloatExpr (FloatType) 86))) (BinopExpr (BoolType) (BinopExpr (BoolType) (FloatExpr (FloatType) 97) == (FloatExpr (FloatType) 82)) == (BinopExpr (BoolType) (FalseExpr (BoolType)) || (TrueExpr (BoolType)))))) (BinopExpr (BoolType) (UnopExpr (BoolType) ! (UnopExpr (BoolType) ! (FalseExpr (BoolType)))) && (BinopExpr (BoolType) (FloatExpr (FloatType) 0) >= (FloatExpr (FloatType) 15))))))
(StructCmd a a (VoidType))
(ShowCmd (BinopExpr (BoolType) (UnopExpr (FloatType) - (DotExpr (FloatType) (StructLiteralExpr (StructType rgba) rgba (UnopExpr (FloatType) - (FloatExpr (FloatType) 96)) (BinopExpr (FloatType) (UnopExpr (FloatType) - (FloatExpr (FloatType) 5)) % (BinopExpr (FloatType) (FloatExpr (FloatType) 12) % (FloatExpr (FloatType) 81))) (UnopExpr (FloatType) - (UnopExpr (FloatType) - (FloatExpr (FloatType) 8))) (FloatExpr (FloatType) 91)) a)) != (FloatExpr (FloatType) 4)))
(StructCmd b a (StructType a))
(StructCmd c a (FloatType) b (FloatType) c (IntType))
(StructCmd d a (IntType))
(StructCmd e a (FloatType) b (StructType c))
(StructCmd f a (VoidType))
(ShowCmd (FloatExpr (FloatType) 91))
(ShowCmd (StructLiteralExpr (StructType e) e (IfExpr (FloatType) (UnopExpr (BoolType) ! (BinopExpr (BoolType) (TrueExpr (BoolType)) && (TrueExpr (BoolType)))) (FloatExpr (FloatType) 94) (UnopExpr (FloatType) - (FloatExpr (FloatType) 41))) (StructLiteralExpr (StructType c) c (FloatExpr (FloatType) 44) (UnopExpr (FloatType) - (BinopExpr (FloatType) (FloatExpr (FloatType) 97) / (FloatExpr (FloatType) 16))) (DotExpr (IntType) (ArrayIndexExpr (StructType d) (ArrayLiteralExpr (ArrayType (StructType d) 1) (StructLiteralExpr (StructType d) d (IntExpr (IntType) 65))) (UnopExpr (IntType) - (UnopExpr (IntType) - (IntExpr (IntType) 371)))) a))))
(ShowCmd (UnopExpr (IntType) - (ArrayIndexExpr (IntType) (ArrayLiteralExpr (ArrayType (IntType) 1) (IntExpr (IntType) 408)) (BinopExpr (IntType) (IfExpr (IntType) (UnopExpr (BoolType) ! (TrueExpr (BoolType))) (UnopExpr (IntType) - (IntExpr (IntType) 93)) (IntExpr (IntType) 884)) + (IntExpr (IntType) 916)))))
(ShowCmd (BinopExpr (BoolType) (BinopExpr (BoolType) (IntExpr (IntType) 436) != (IfExpr (IntType) (TrueExpr (BoolType)) (IntExpr (IntType) 774) (IntExpr (IntType) 774))) && (BinopExpr (BoolType) (IntExpr (IntType) 681) <= (DotExpr (IntType) (StructLiteralExpr (StructType d) d (IntExpr (IntType) 223)) a))))
(ShowCmd (BinopExpr (BoolType) (BinopExpr (FloatType) (DotExpr (FloatType) (StructLiteralExpr (StructType e) e (FloatExpr (FloatType) 48) (StructLiteralExpr (StructType c) c (UnopExpr (FloatType) - (FloatExpr (FloatType) 26)) (BinopExpr (FloatType) (FloatExpr (FloatType) 21) % (FloatExpr (FloatType) 8)) (IntExpr (IntType) 648))) a) + (UnopExpr (FloatType) - (FloatExpr (FloatType) 54))) <= (FloatExpr (FloatType) 55)))
